exercise/templates/exercise/staff/results.html

```diff
@@ -10,45 +11,185 @@
+                <input type="checkbox" id="max-p-checkbox" value="max-p">{% trans "Max points" %}
+            </label>
+        </div>
+
+
+
+        <!--Tags-->
+        <div style="margin: 10px 10px 10px 20px;" class="filter-users">
+            <small>{% trans "Filter users" %}:</small>
+            <button class="btn btn-success btn-xs" data-id="aalto">
+              <span class="glyphicon glyphicon-unchecked" aria-hidden="true"></span>
+              {{ internal_user_label|safe }}
+            </button>
+            <button class="btn btn-info btn-xs" data-id="mooc">
+              <span class="glyphicon glyphicon-unchecked" aria-hidden="true"></span>
+              {{ external_user_label|safe }}
+            </button>
+            {% for tag in tags %}
+            <button class="btn btn-default btn-xs" style="background-color:{{ tag.color }};color:{{ tag.font_color }};" data-id="{{ tag.id }}">
+              <span class="glyphicon glyphicon-unchecked" aria-hidden="true"></span>
+              {{ tag.name }}
+            </button>
+            {% endfor %}
+        </div>
+
+
+        <!--Navigation bar to choose how data is shown-->
+        <ul style="margin: 10px 0px 50px 0px;" id="exercise-nav-tabs" class="nav nav-tabs">
+            <li id="difficulty-exercises" role="presentation" class="active">
+                <a onclick="createPointTable('difficulty');" data-toggle="tab" href="#">{% trans "Show exercises by difficulty" %}</a>
+            </li>
+            <li id="module-exercises" role="presentation">
+                <a onclick="createPointTable('module');" data-toggle="tab" href="#">{% trans "Show exercises by module" %}</a>
+            </li>
+            <li id="all-exercises" role="presentation">
+                <a onclick="createPointTable('all');" data-toggle="tab" href="#">{% trans "Show all exercises" %}</a>
+            </li>
+        </ul>
+
+        <!--Dropdown button for table exports. The buttons are added inside results_staff.js-->
+        <div class="dropdown d-inline-block">
+            <button class="btn btn-primary dropdown-toggle" type="button" data-toggle="dropdown">{% trans "Export Data" %}
+                  <span class="caret"></span></button>
+            <ul id="export-button-menu" class="dropdown-menu">
+
+            </ul>
+        </div>
+
+    </div>
+
+        <!--Data table-->
+        <div style="display: none;" id="table-points-div" class="table-responsive">
+            <table id="table-points" class="table table-striped table-bordered table-condensed filtered-table">
+                <thead id="table-heading">
+                    <tr id="table-heading-row">
+                    </tr>
+                </thead>
+                <tbody id="table-body">
+
+                </tbody>
+            </table>
+        </div>
+
+            <!--Loading animation-->
+    <div class="row">
+        <div class="col-md-12">
+            <div id="loading-animation" class="loader">
```

Comment:
The class is probably a bit generic (`loader`)? Also, can we use the same css for this and exercise loading? Should they have the same? Should we actually use this one for the exercises?

Reply:
changed the name to loading-animation. Did not do anything else yet. Not sure what would be good option.